src/components/addSourceWizard/compileAllApplicationComboOptions.js

```diff
@@ -54,23 +47,12 @@ export const labelMapper = (type, intl, hcsEnrolled) =>
     ),
   })[type.name];
 
-export const compileAllApplicationComboOptions = (applicationTypes, intl, activeCategory, hcsEnrolled) => [
+export const compileAllApplicationComboOptions = (applicationTypes, intl, hcsEnrolled) => [
```

Comment:
the params change needs to be reflected across all usages of this function for the tests to pass